SConstruct

```diff
@@ -119,13 +120,29 @@ opts.Add("p", "Platform (alias for 'platform')", "")
 opts.Add("platform", "Target platform (%s)" % ("|".join(platform_list),), "")
 opts.Add(BoolVariable("tools", "Build the tools (a.k.a. the Godot editor)", True))
 opts.Add(EnumVariable("target", "Compilation target", "debug", ("debug", "release_debug", "release")))
-opts.Add("arch", "Platform-dependent architecture (arm/arm64/x86/x64/mips/...)", "")
-opts.Add(EnumVariable("bits", "Target platform bits", "default", ("default", "32", "64")))
 opts.Add(EnumVariable("float", "Floating-point precision", "default", ("default", "32", "64")))
 opts.Add(EnumVariable("optimize", "Optimization type", "speed", ("speed", "size", "none")))
 opts.Add(BoolVariable("production", "Set defaults to build Godot for use in production", False))
 opts.Add(BoolVariable("use_lto", "Use link-time optimization", False))
 
+# CPU architecture options.
+architectures = ["", "x86_32", "x86_64", "arm32", "arm64", "rv64", "ppc32", "ppc64", "wasm32"]
```

Comment:
For `x86` and `arm` architectures, do we really need to change their nomenclature within the build configs?

Especially as `x86` usually indicates 32 bits, while `x86_64` indicates 64 bits. Similarly for `arm`, `armeabi-v7a` indicates 32 bits and `arm64-v8a` indicates 64 bits.

Seems like we can just use the same terms as used by everyone else instead of coming up with our own (`arm32`, `arm64`) terms.

Reply:
Yes, we should be explicit. 32-bit versions of these architectures are getting to be more and more niche and it's common to refer to both 32-bit and 64-bit versions of x86 as "x86". We want to avoid people wanting to build "for x86" or "for ARM" and accidentally building for what-will-be-ancient 32-bit versions of them. Also, these terms are not dominant but they are not unique to Godot either, I've seen many usages of "x86_32" online.